### DISCUSSION TITLE
encountered an unknown problem

### DISCUSSION THREAD

**Discussion by zayn-0101** [vercel/community#7338] [Account Issue]
### Question

I can't deploy any project, and the page shows the massage that "Your Team encountered an unknown problem. Please reach out to our support team for details"
Please help me with this, my Vercel ID is: aJkwMxDtnDeJDRhT2xCJug5i

**Comment by pawlean** [comment]
Hi @zion101,

You should be able to access it now! Let me know if we help with anything else.

**Comment by zayn-0101** [comment]
Yes!! Thanks so much for your help
*Johnny*


Pauline P. Narvas ***@***.***> 于2024年7月1日周一 18:39写道：

**Reply by pawlean** [reply]
No worries, Johnny! Let me know if you have any more questions.

**Comment by taanng** [comment]
@pawlean  i have the same problem , pls help me , my Vercel ID is keQDCrMICV0Bxe3wdxYVIvE9

**Reply by amyegan** [reply]
Hey @taanng! It should work for you now.

Please join us in the new [Vercel Community](https://vercel.community) if there's anything you want to discuss in the future 😄
